verifier/src/lib.rs

```diff
@@ -43,7 +43,8 @@ pub fn verify(
 ) -> Result<(), VerificationError> {
     // build public inputs and try to verify the proof
     let pub_inputs = PublicInputs::new(program_info, stack_inputs, stack_outputs);
-    winterfell::verify::<ProcessorAir>(proof, pub_inputs).map_err(VerificationError::VerifierError)
+    winterfell::verify::<ProcessorAir, Rpo256>(proof, pub_inputs)
+        .map_err(VerificationError::VerifierError)
```

Comment:
This would still need to be `BLAKE3` for now.

Reply:
Done!